jobs/eventsRetriever.test.ts

```diff
@@ -30,7 +30,7 @@ describe('test', () => {
 
     const call = welcomeClient.getEventsCSV.mock.calls[0]
     expect(dataStartDate.isSame(call[1])).toBe(true)
-    expect(call[2]).toBe(365)
+    expect(call[2]).toBe(183)
```

Comment:
Will 183 always be correct? wary that this test may fail next month

Reply:
Sorry, just seen that this `183` is hard coded in the function below

Reply:
I used 183 to account for leap years. Essentially to code will look at the period from 6 months back to today and fetch all the wpip moves in that period.  If the report was being requested on 29 Feb the 183rd day will capture it.